pkg/config/parameters_test.go

```diff
@@ -390,15 +393,22 @@ tls:
 `)
 
 	check(func(t *testing.T, conf *Parameters) {
-		assert.Equal(t, "1.3", conf.TLS.MinimumProtocolVersion)
+		assert.Equal(t, "1.2", conf.TLS.MinimumProtocolVersion)
+		assert.Equal(t, "1.3", conf.TLS.MaximumProtocolVersion)
 		assert.Equal(t, TLSCiphers{"ECDHE-RSA-AES256-GCM-SHA384"}, conf.TLS.CipherSuites)
 	}, `
 tls:
-  minimum-protocol-version: 1.3
+  minimum-protocol-version: 1.2
+  maximum-protocol-version: 1.3
   cipher-suites:
   - ECDHE-RSA-AES256-GCM-SHA384
 `)
 
+	checkFail(`
```

Comment:
for full coverage looks like we can add a couple cases where min/max tls protocols provided are invalid